File: WebAdmin/Models/OpeningHours.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WebAdmin.Models
{
    public class OpeningHours
    {
        public int WeekDay {get; set;}
        public DateTime OpeningTime {get ;set;}
        public DateTime ClosingTime {get; set;}
    }
}

File: WebAdmin/src/Models/OpeningHours.js
class OpeningHours {
    WeekDay;
    OpeningTime;
    ClosingTime;

    constructor(weekDay = 0, openingTime = '', closingTime = ''){
        this.WeekDay = weekDay;
        this.OpeningTime = openingTime;
        this.ClosingTime = closingTime;
    }
}